### PR TITLE
Add logging of the latest ping latencies

### PR DESCRIPTION
**Pull request by xosmig**

### CODE REVIEW

**Review by bchocho — APPROVED** [review]
LGTM

File: peer-monitoring-service/client/src/peer_states/latency_info.rs [hunk 1]
```diff
@@ -109,6 +109,14 @@ impl LatencyInfoState {
         }
     }
 
+    /// Returns the latest latency ping in seconds. If no latency
+    /// pings have been recorded, None is returned.
+    pub fn get_latest_latency_ping_secs(&self) -> Option<f64> {
+        self.recorded_latency_ping_durations_secs
+            .last_key_value()
+            .map(|(&_key, &value)| value)
```
Comment (JoshLind) [inline peer-monitoring-service/client/src/peer_states/latency_info.rs:117]: nit: this might look neater as `.map(|_, value| *value)`?

**Review by JoshLind — APPROVED** [review]
LGTM! 😄

File: peer-monitoring-service/client/src/peer_states/latency_info.rs (hunk 1 above)
Comment (xosmig) [reply peer-monitoring-service/client/src/peer_states/latency_info.rs]: fixed, thanks!

**Pull request merged**